protocol/x/sending/keeper/msg_server_create_transfer.go

```diff
@@ -97,7 +100,7 @@ func (k msgServer) WithdrawFromSubaccount(
 	return &types.MsgWithdrawFromSubaccountResponse{}, nil
 }
 
-// SendFromModuleToAccount sends a coin from a module to an account.
+// SendFromModuleToAccount sends coins from a module to an account.
```

Comment:
To confirm: (I'm not super familiar with x/sender) - this is to an account and not a subaccount?

Reply:
(Is this also an x/bank recipient?)

Reply:
yep this is an x/bank recipient